t/fuzzing/serverless_route_test.py

```diff
@@ -100,12 +92,10 @@ def main():
     # before test
```

Comment:
Can we refactor this part as function and use `create_route` & `main` as arguments?

Reply:
I got it.